README.md

```diff
@@ -1,111 +1,77 @@
 # David
-An engine for chatbots. 
+
+[![Documentation Status](https://readthedocs.org/projects/david-ausubel/badge/?version=latest)](https://david-ausubel.readthedocs.io/en/latest/?badge=latest)
+
+
+**David** is a powerfull engine dialogue engine with two main parts: `NLU` and `engine`. The main objetive of this project is too use existing chatbots projects and uses it to develop your solution.
```

Comment:
What do you think of calling the second part of the engine of DIALOGUE?

Reply:
I think it is better! Rasa call it `core` but `dialogue` makes more sense to me.